plugin.video.montreal.greek-tv/resources/lib/default.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+import urllib.parse, os, sys
+import xbmcvfs, xbmcgui, xbmcplugin, xbmcaddon
+
+addon_id       = 'plugin.video.montreal.greek-tv'
+Home           = xbmcvfs.translatePath(xbmcaddon.Addon().getAddonInfo('path'))
+art            = xbmcvfs.translatePath(Home + '/resources/art/')
```

Comment:
Good practice to use os.path.join instead of concatenating string paths. use os.path.join(Home, "resources","art")